jupyter/datascience/anaconda-python-3.8/environment.yml

```diff
@@ -0,0 +1,356 @@
+  - decorator=5.1.1=pyhd8ed1ab_0
+  - defusedxml=0.7.1=pyhd8ed1ab_0
+  - deprecated=1.2.13=pyh6c4a22f_0
+  - deprecation=2.1.0=pyh9f0ad1d_0
+  - dill=0.3.6=pyhd8ed1ab_1
+  - docstring-to-markdown=0.12=pyhd8ed1ab_0
+  - docstring_parser=0.15=pyhd8ed1ab_0
+  - elyra=3.15.0=pyhd8ed1ab_0
+  - entrypoints=0.4=pyhd8ed1ab_0
+  - executing=1.2.0=pyhd8ed1ab_0
+  - expat=2.2.10=h9c3ff4c_0
+  - fire=0.5.0=pyhd8ed1ab_0
+  - flake8=6.0.0=pyhd8ed1ab_0
+  - flatbuffers=2.0.0=h2531618_0
+  - flit-core=3.9.0=pyhd8ed1ab_0
+  - fontconfig=2.14.1=hef1e5e3_0
+  - fonttools=4.25.0=pyhd3eb1b0_0
+  - freetype=2.10.4=h0708190_1
+  - frozenlist=1.3.3=py38h5eee18b_0
+  - future=0.18.3=pyhd8ed1ab_0
+  - gast=0.4.0=pyh9f0ad1d_0
+  - giflib=5.2.1=h36c2ea0_2
+  - gitdb=4.0.10=pyhd8ed1ab_0
+  - gitpython=3.1.31=pyhd8ed1ab_0
+  - glib=2.69.1=he621ea3_2
+  - google-api-core=2.10.0=pyhd8ed1ab_0
+  - google-api-python-client=1.7.12=pyh8c360ce_0
+  - google-auth=2.17.3=pyh1a96a4e_0
+  - google-auth-httplib2=0.1.0=pyhd8ed1ab_1
+  - google-auth-oauthlib=1.0.0=pyhd8ed1ab_0
+  - google-cloud-core=2.3.2=pyhd8ed1ab_0
+  - google-cloud-storage=1.43.0=pyh6c4a22f_0
+  - google-crc32c=1.1.2=py38h57c428a_3
+  - google-pasta=0.2.0=pyh8c360ce_0
+  - google-resumable-media=2.5.0=pyhd8ed1ab_0
+  - googleapis-common-protos=1.57.0=py38h578d9bd_0
+  - grpc-cpp=1.48.2=h5bf31a4_0
+  - grpcio=1.48.2=py38h5bf31a4_0
+  - gst-plugins-base=1.14.1=h6a678d5_1
+  - gstreamer=1.14.1=h5eee18b_1
+  - h5py=3.1.0=nompi_py38hafa665b_100
+  - hdf5=1.10.6=nompi_h6a2412b_1114
+  - httplib2=0.22.0=pyhd8ed1ab_0
+  - icu=58.2=hf484d3e_1000
+  - idna=3.4=pyhd8ed1ab_0
+  - importlib-metadata=6.6.0=pyha770c72_0
+  - importlib_resources=5.12.0=pyhd8ed1ab_0
+  - intel-openmp=2021.4.0=h06a4308_3561
+  - ipykernel=5.5.5=py38hd0cf306_0
+  - ipython=8.4.0=py38h578d9bd_0
+  - ipython_genutils=0.2.0=py_1
+  - ipywidgets=8.0.6=pyhd8ed1ab_0
+  - isort=5.12.0=pyhd8ed1ab_1
+  - jedi=0.18.2=pyhd8ed1ab_0
+  - jinja2=3.1.2=pyhd8ed1ab_1
+  - joblib=1.2.0=pyhd8ed1ab_0
+  - jpeg=9e=h166bdaf_1
+  - json5=0.9.5=pyh9f0ad1d_0
+  - jsonschema=3.2.0=pyhd8ed1ab_3
+  - jupyter-lsp=2.1.0=pyhd8ed1ab_0
+  - jupyter-packaging=0.12.3=pyha770c72_1
+  - jupyter-resource-usage=0.7.1=pyhd8ed1ab_0
+  - jupyter-server-mathjax=0.2.6=pyh5bfe37b_1
+  - jupyter-server-proxy=4.0.0=pyhd8ed1ab_0
+  - jupyter_bokeh=3.0.7=pyhd8ed1ab_0
+  - jupyter_client=7.0.6=pyhd8ed1ab_0
+  - jupyter_contrib_core=0.4.0=pyhd8ed1ab_0
+  - jupyter_contrib_nbextensions=0.7.0=pyhd8ed1ab_0
+  - jupyter_core=5.3.0=py38h578d9bd_0
+  - jupyter_events=0.6.3=pyhd8ed1ab_0
+  - jupyter_highlight_selected_word=0.2.0=py38h578d9bd_1005
+  - jupyter_kernel_gateway=2.5.2=pyhd8ed1ab_1
+  - jupyter_latex_envs=1.4.6=pyhd8ed1ab_1002
+  - jupyter_nbextensions_configurator=0.6.1=pyhd8ed1ab_0
+  - jupyter_server=1.23.6=pyhd8ed1ab_0
+  - jupyter_server_terminals=0.4.4=pyhd8ed1ab_1
+  - jupyterlab=3.5.3=pyhd8ed1ab_0
+  - jupyterlab-git=0.41.0=pyhd8ed1ab_1
+  - jupyterlab-lsp=3.10.2=pyhd8ed1ab_0
```

Comment:
Seems like we are not including the `jupyterlab-s3-browser` plugin via environment.yml
is there some specific reason ?

Reply:
No equivalent Anaconda package, there's a random third party one with 116 downloads, didn't feel like a good option